src/otx/recipe/_base_/data/anomaly.yaml

```diff
@@ -13,11 +13,10 @@ train_subset:
   batch_size: 32
   num_workers: 4
   transforms:
-    - class_path: otx.core.data.transform_libs.torchvision.ResizetoLongestEdge
+    - class_path: torchvision.transforms.v2.Resize
       init_args:
-        size: $(input_size)
+        size: [256, 256]
```

Comment:
```
size: $(input_size)
```

Reply:
When I use `$(input_size)` it resizes only along one dimension. And, when I pass `$(input_size), $(input_size)`, I get `found str expected int` error.

Reply:
if you want to use list as input size then you need to set input size as list in yaml file as my comment above.